airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SignupForm.tsx

```diff
@@ -0,0 +1,231 @@
+import { Field, FieldProps, Formik } from "formik";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import styled from "styled-components";
+import * as yup from "yup";
+
+import { LabeledInput, Link, LoadingButton } from "components";
+
+import { useConfig } from "config";
+import { FieldError } from "packages/cloud/lib/errors/FieldError";
+import { useAuthService } from "packages/cloud/services/auth/AuthService";
+
+import CheckBoxControl from "../../components/CheckBoxControl";
+import { BottomBlock, FieldItem, Form, RowFieldItem } from "../../components/FormComponents";
+
+interface FormValues {
+  name: string;
+  companyName: string;
+  email: string;
+  password: string;
+  news: boolean;
+  security: boolean;
+}
+
+const SignupPageValidationSchema = yup.object().shape({
+  email: yup.string().email("form.email.error").required("form.empty.error"),
+  password: yup.string().min(12, "signup.password.minLength").required("form.empty.error"),
+  name: yup.string().required("form.empty.error"),
+  companyName: yup.string().required("form.empty.error"),
+  security: yup.boolean().oneOf([true], "form.empty.error"),
+});
+
+const MarginBlock = styled.div`
```

Comment:
I'm assuming this is touched by our sign in page redesign, so makes sense to migrate to sass with that change instead.

Reply:
Correct. I added this because it was needed but my intention was not to introduce SCSS into this PR.